frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Der Name sollte angepasst werden (Für Prototyp nicht wichtig)

Reply:
Ich muss sagen ich hab in den `public` Ordner gar nicht reingeguckt. Die Logos dort könnten auch raus bzw. ersetzt werden.

Reply:
Alles gut^^ Soll ja erstmal grundsätzlich funktionieren. Das sind ja alles Kleinigkeiten die schnell erledigt werden können.